Deployment_files/test-3_kube-pod

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
```

Comment:
no pods just deployments , replicasets or statefulsets.......

Reply:
Hi Audun!

Have you tried making a pull request?

Reply:
Please revert the PR, it causes instability to the main branch!!!

Reply:
I will forward the request to our QA department